### ISSUE TITLE
Use PR target branch head as the base commit

### ISSUE THREAD

**Issue by jouni** [jitterbit/get-changed-files#17]
Is it possible to get a list of changed files between a pull request and the branch it targets? Currently the action returns the changes between commits within the same PR.

I would like to avoid the need to squash commits within the PR during code review, but still be able to get a list of all the changed files in the PR branch.

**Comment by johentsch** [comment]
Yes, I would also love to see this. Currently when I copy a development branch and launch a PR, the base commit is only 0's.
